client/my-sites/comments/comment/index.jsx

```diff
@@ -210,13 +255,16 @@ const mapStateToProps = ( state, { commentId } ) => {
 	const siteId = getSelectedSiteId( state );
 	const comment = getSiteComment( state, siteId, commentId );
 	const commentStatus = get( comment, 'status' );
+	const currentUserId = getCurrentUserId( state );
 	return {
 		siteId,
 		postId: get( comment, 'post.ID' ),
 		commentIsPending: 'unapproved' === commentStatus,
+		hasRepliedToComment: ! get( comment, 'i_replied' ),
```

Comment:
I should have more time to take a closer look and test in more depth on Monday, @Aurorum, but I am a bit confused by the naming/logic here. I think `comment.i_replied` should be `true` when I have replied to the comment, so using `! get( comment, 'i_replied' )` to compute `hasRepliedToComment` feels like it's wrong. Can we work around that to use either `!!` or name the prop `commentHasNoReply` (or event `commentHasReply` with the flipped logic).

Reply:
Thanks so much for taking a look! Yes, you're absolutely right - that was a typo. I've fixed it now. :)